crates/shaders/Cargo.toml

```
@@ -1,10 +1,14 @@
 [package]
 name = "vello_shaders"
-version = "0.1.0"
+version.workspace = true # We mimic Vello's version
+description = "Vello infrastructure to compile pipelines and shaders."
```

I don't think vello_shaders handles pipeline compilation. That's handled in whatever integration uses `vello_shaders`.

That is,
```suggestion
description = "Vello infrastructure to preprocess and cross-compile shaders at compile time"
```

---

This is another one I got from `ARCHITECTURE.md`. I updated the changes here, and will also submit a comment on the docs PR so that we're more precise there too.